CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.15.0-rc.0 / 2018-02-28
+
+* [CHANGE] Moved to memberlist as underlying gossip protocol
+* [ENHANCEMENT] Support receiving alerts with end time and zero start time
```

Comment:
An additional enhancement is the ability to configure the HTTP clients (see https://github.com/prometheus/alertmanager/commit/d678022fea943108b4fbeece906428ab23d5ccc5).